doc/ug/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+file(GLOB ug_TEXFILES RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} *.tex)
+file(GLOB ug_BIBFILES RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} *.bib)
+file(GLOB ug_FIGS RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} figures/*.pdf)
+file(GLOB ug_TEXFILES_abs *tex)
+
+find_program(AWK awk)
+
+add_custom_command(OUTPUT quickref.inp
+                   COMMAND ${AWK} -f
+                   ${CMAKE_CURRENT_SOURCE_DIR}/assemble_quickref.awk
+                   ${ug_TEXFILES_abs} > ${CMAKE_CURRENT_BINARY_DIR}/quickref.inp
+)
+
+add_custom_target(ug
+        COMMAND ${CMAKE_BINARY_DIR}/doc/latexit.sh
+        ${CMAKE_CURRENT_SOURCE_DIR} ug
+        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
```

Comment:
Don't need `${CMAKE_CURRENT_BINARY_DIR}` here, `add_custom_target`'s `WORKING_DIRECTORY` is relative to the current binary directory, so either use `.` or leave out the parameter.